bin/main.rs

```diff
@@ -12,64 +22,81 @@ use thiserror::Error;
 mod bind;
 mod init;
 
+/// Represents command-line arguments passed to the `Arbiter` tool.
 #[derive(Parser)]
 #[command(name = "Arbiter")]
 #[command(version = env!("CARGO_PKG_VERSION"))]
-#[command(about = "Data analysis tool for decentralized exchanges.", long_about = None)]
+#[command(about = "Ethereum Virtual Machine Logic Simulator", long_about = None)]
```

Comment:
This reminds me that we should go through and look for things like this and clean up our README and Github page in general.

Reply:
Yeah 100%